# HG changeset patch
# User joe
# Date 1537549064 25200
#      Fri Sep 21 09:57:44 2018 -0700
# Node ID bef40b39c6d126aff9c4dfe8e3ffe8420f96e92e
# Parent  9a8f28029c768d0e4f190a64f43bc4e5a4cbeb50
switch the background and foreground colours in sam

diff -r 9a8f28029c76 -r bef40b39c6d1 sys/src/cmd/samterm/flayer.c
--- a/sys/src/cmd/samterm/flayer.c	Fri Sep 21 09:00:03 2018 -0700
+++ b/sys/src/cmd/samterm/flayer.c	Fri Sep 21 09:57:44 2018 -0700
@@ -29,17 +29,17 @@
 	lDrect = r;
 
 	/* Main text is yellowish */
-	maincols[BACK] = allocimagemix(display, DPaleyellow, DWhite);
+	maincols[BACK] = display->black /* allocimagemix(display, DPaleyellow, DWhite)*/;
 	maincols[HIGH] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DDarkyellow);
 	maincols[BORD] = allocimage(display, Rect(0,0,2,2), screen->chan, 1, DYellowgreen);
-	maincols[TEXT] = display->black;
+	maincols[TEXT] = allocimagemix(display, DPaleyellow, DWhite)/* display->black*/;
 	maincols[HTEXT] = display->black;
 
 	/* Command text is blueish */
-	cmdcols[BACK] = allocimagemix(display, DPalebluegreen, DWhite);
+	cmdcols[BACK] = display->black /* allocimagemix(display, DPalebluegreen, DWhite)*/;
 	cmdcols[HIGH] = allocimage(display, Rect(0,0,1,1), screen->chan, 1, DPalegreygreen);
 	cmdcols[BORD] = allocimage(display, Rect(0,0,2,2), screen->chan, 1, DPurpleblue);
-	cmdcols[TEXT] = display->black;
+	cmdcols[TEXT] = allocimagemix(display, DPalebluegreen, DWhite) /* display->black*/;
 	cmdcols[HTEXT] = display->black;
 }
 
